webapp/packages/core-blocks/src/FormControls/Textarea.tsx

```diff
@@ -102,12 +103,14 @@ export const Textarea: TextareaType = observer(function Textarea({
 
   return (
     <Field {...layoutProps} className={s(styles, { field: true, embedded }, className)}>
-      <FieldLabel className={s(styles, { fieldLabel: true })} title={labelTooltip || rest.title} required={required}>
+      <FieldLabel htmlFor={inputId} className={s(styles, { fieldLabel: true })} title={labelTooltip || rest.title} required={required}>
         {children}
       </FieldLabel>
       <textarea
         {...rest}
         ref={textareaRef}
+        id={inputId}
+        aria-label={children?.toString()}
```

Comment:
Should we change the type of children to "string"?

Reply:
same as for input, but in textarea case, the situation is even worse, because in spec, textarea's children = value.